src/components/menus/GameMenu.js

```diff
@@ -24,10 +23,11 @@ export default function GameMenu() {
 
   return (
     <>
-      {isModalOpen && <Modal onClose={handleModalClose} message={message} />}
+      {isModalOpen && (
+        <Modal handleModalClose={handleModalClose} message={message} />
```

Comment:
각 버튼의 핸들러 이름이 handle ~ Button Click 인 것처럼, handleModalClick의 네이밍이 어떨까요?
```suggestion
        <Modal handleModalClick={handleModalClose} message={message} />
```

Reply:
제안 감사합니다. 그런데 모달창을 클릭 했을 때 handleModalClose 함수는 모달창을 닫게 하는 역할이라서 제안은 감사하지만, 변수명은 그대로 두는게 혼란을 줄일 수 있을 것 같습니다 :)